exonum-java-binding/core/rust/ejb-app/src/main.rs

```diff
@@ -25,6 +26,7 @@ extern crate tempfile;
 mod node_builder;
 
 fn main() {
+    env_logger::init();
```

Comment:
What is its configuration?

Reply:
Configuration is done using RUST_LOG env var: https://docs.rs/env_logger/0.6.1/env_logger/